releasenotes/notes/cgroup2-a3868da0e9ae5450.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    APM - Add support for cgroup2 via UDS.
```

```suggestion
    APM: Add support for cgroup2 via UDS.
```
We might need a bit more details here. What is the actual user impact? How does it enhance the experience now? In which cases is the container ID now retrieved whereas it previously wasn't?